README.md

```diff
@@ -7,13 +7,11 @@ This is a [Viam module](https://docs.viam.com/extend/modular-resources/) providi
 
 ## Getting started
 
-To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/modular-resources/configure/#add-a-module-from-the-viam-registry) and select the `viam:vision:re-id-object-tracker` model from the [`re-id-object-tracker` module](https://app.viam.com/module/re-id-object-tracker).
 This module implements the following methods of the [vision service API](https://docs.viam.com/services/vision/#api).
```

Comment:
I think this is fine to keep as additional info
```suggestion
To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/modular-resources/#configuration) and select the `viam:vision:re-id-object-tracker` model from the [`re-id-object-tracker` module](https://app.viam.com/module/re-id-object-tracker).
This module implements the following methods of the [vision service API](https://docs.viam.com/services/vision/#api):
```

Reply:
gotcha, was thinking duplication